lfsapi/ntlm.go

```diff
@@ -144,7 +144,7 @@ func ntlmGetCredentials(creds creds.Creds) (*ntmlCredentials, error) {
 
 	splits := strings.Split(username, "\\")
 	if len(splits) != 2 {
-		return nil, fmt.Errorf("Your user name must be of the form DOMAIN\\user. It is currently '%s'", username)
+		return nil, fmt.Errorf("your user name must be of the form DOMAIN\\user. It is currently '%s'", username)
```

Comment:
I think this one and the one above are intended to be sentences, and so should begin with a capital.

Reply:
Ok, reverted